tools/LoadPackage.groovy

```diff
@@ -96,7 +96,7 @@ class LoadPackage {
 
     def run(File package_dir) {
```

Comment:
prefer "void run" since it does not return any object .. and check the other signatures ..(http://groovy-lang.org/style-guide.html -> "When using def in Groovy, the actual type holder is Object")